### PR TITLE
Enterprise Coupon ADR

### PR DESCRIPTION
**Pull request by brittneyexline**

### CODE REVIEW

**Review by christopappas — APPROVED** [review]
I'm not sure if there is a specific rubric for approving an ADR, but it was an informative read, and looks to be a great summary of all the work done, so 👍 from me

**Review by muhammad-ammar — APPROVED** [review]
Looks good to me.

@georgebabey It would be great if you can also take a look this.

File: docs/decisions/0003-enterprise-coupons.rst [hunk 1]
```diff
@@ -0,0 +1,151 @@
+3. Enterprise Coupons
+---------------------
+
+Status
+------
+
+Accepted?
```
Comment (georgebabey) [inline docs/decisions/0003-enterprise-coupons.rst]: `Accepted` once it merges!

**Review by georgebabey — APPROVED** [review]
Looks great, thanks for getting this written up!

**Pull request merged**